BetSystem/Model/BetOnEvent.cs

```diff
@@ -4,11 +4,16 @@ public class BetOnEvent
     {
 
         public int Id { get; set; }
-        public SportEvent Event { get; set; }
-        public Team Team { get; set; }
+        public SportEvent Event { get; set; } = new SportEvent();
+        public Team Team { get; set; } = new Team();
         public BetResult BetOnResult { get; set; }
         public int Amount { get; set; }
         public int Interest { get; set; }
         public int AmountToPay { get; set; }
+
+        internal void UpdateAmountToPay(EventResult eventResult)
```

Comment:
I think in DB models we also don't implement logic.

Reply:
UpdateAmountToPay implemented in SportEventService